lib/index.ts

```diff
@@ -0,0 +1,468 @@
+        fs.mkdirSync(this.options.tempDir);
+      }
+      fs.mkdirSync(this.uuid);
+      fs.mkdirSync(path.resolve(this.uuid, './OEBPS'));
+      if (!this.options.css) {
+        this.options.css = fs.readFileSync(path.resolve(__dirname, '../templates/template.css'), { encoding: 'utf-8' });
+      }
+      fs.writeFileSync(path.resolve(this.uuid, './OEBPS/style.css'), this.options.css);
+      if (this.options.fonts.length) {
+        fs.mkdirSync(path.resolve(this.uuid, './OEBPS/fonts'));
+        this.options.fonts = this.options.fonts.map((font) => {
+          if (!fs.existsSync(font)) {
+            reject(new Error('Custom font not found at ' + font + '.'));
+            return;
+          }
+          const filename = path.basename(font);
+          fs.copyFileSync(font, path.resolve(this.uuid, './OEBPS/fonts/' + filename));
+          return filename;
+        });
+      }
+      this.options.content.forEach((content) => {
+        let data = `${this.options.docHeader}
+        <head>
+        <meta charset="UTF-8" />
+        <title>${entities.encodeXML(content.title || '')}</title>
+        <link rel="stylesheet" type="text/css" href="style.css" />
+        </head>
+        <body>\
+`;
+        data +=
+          content.title && this.options.appendChapterTitles
+            ? `<h1>${entities.encodeXML(content.title)}</h1>`
+            : "";
+        data +=
+          content.title && content.author && content.author.length
+            ? `<p class='epub-author'>${entities.encodeXML(
+              Array.isArray(content.author) ? content.author.join(", ") : content.author
+            )}</p>`
+            : "";
+        data +=
+          content.title && content.url
+            ? `<p class='epub-link'><a href='${content.url}'>${content.url}</a></p>`
+            : "";
+        data += `${content.data}</body></html>`;
+        return fs.writeFileSync(content.filePath, data);
+      });
+
+      // write meta-inf/container.xml
+      fs.mkdirSync(this.uuid + '/META-INF');
+      fs.writeFileSync(
+        `${this.uuid}/META-INF/container.xml`,
+        '<?xml version="1.0" encoding="UTF-8" ?><container version="1.0" xmlns="urn:oasis:names:tc:opendocument:xmlns:container"><rootfiles><rootfile full-path="OEBPS/content.opf" media-type="application/oebps-package+xml"/></rootfiles></container>'
+      );
+
+      if (this.options.version === 2) {
+        // write meta-inf/com.apple.ibooks.display-options.xml [from pedrosanta:xhtml#6]
+        fs.writeFileSync(
+          `${this.uuid}/META-INF/com.apple.ibooks.display-options.xml`,
+          `\
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<display_options>
+  <platform name="*">
+    <option name="specified-fonts">true</option>
+  </platform>
+</display_options>\
+`
+        );
+      }
+
+      const opfPathTs = path.resolve(
+        __dirname,
+        `../templates2/epub${this.options.version}/content.opf.ts`
+      );
+      if (!fs.existsSync(opfPathTs)) {
```

Comment:
Just import both templates at the top

`import {createContentOpfStr} from '../templates2/...'`

No need for async imports.

Reply:
well, this is due to version epub selection ( there's same named exports in both `epub2`, `epub3` files - hence the dynamic import usage)